waiter/src/waiter/scheduler.clj

```diff
@@ -47,6 +47,11 @@
   [& args]
   `(log/log "Scheduler" :debug nil (print-str ~@args)))
 
+(defmacro log-service-instance
```

Comment:
Can we have this as a function instead of a macro?

I do not expect instance logging events to be that frequent. In any case, they are not on the reserve instance hot paths.

Reply:
Done